specifyweb/frontend/js_src/lib/components/FormParse/webOnlyViews.ts

```diff
@@ -84,6 +84,37 @@ export const webOnlyViews = f.store(() =>
       'edit',
       ['name']
     ),
+    [adminUser]: autoGenerateViewDefinition(
+      tables.SpecifyUser,
+      'form',
+      'edit',
+      ['name', 'password', 'agents']
+    ),
+    [institution]: autoGenerateViewDefinition(
+      tables.Institution,
+      'form',
+      'edit',
+      ['name', 'code', 'isAccessionsGlobal', 'isSingleGeographyTree', 'isSecurityOn', 'isServerBased']
```

Comment:
`isSecurityOn` and `isServerBased` should probably be set automatically and not input by the user.

`IsAccessionsGlobal` and `IsSingleGeographyTree` will affect application behavior and absolutely impact the later generation of disciplines (each discipline would not get the choice to create a geography tree)